www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php

```diff
@@ -29,13 +29,11 @@
  */
 
 require_once("config.inc");
-require_once("certs.inc");
-require_once("legacy_bindings.inc");
```

Comment:
I'm not 100% sure about dropping this.

Also, you have a problem with the permission restriction because you write the file first and only then change the permission in extract_and_save_certificates()

Reply:
I restored the certs.inc and legacy_bindings.inc. I thought I didn't need them since I don't use them in any function inside the script. The only hard requirement was config.inc since "use OPNsense\Core\Config;" depends on it.

Additionally, I removed the chmod in the php script, and changed the permission of the folder the certificates are saved in, so they are 0600 the moment they are saved, rather than afterwards.

I hope that's sufficient, since I want to avoid having to use something like "umask" if possible.